<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imageshow</class>
 <widget class="QWidget" name="imageshow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_imagesave">
         <property name="text">
          <string>保存图片</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_imageread">
         <property name="text">
          <string>导入图片</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_videosave">
         <property name="text">
          <string>导出视频</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_videoread">
         <property name="text">
          <string>导入视频</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_videoshow">
         <property name="text">
          <string>开始播放</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>图像处理</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="Pic_01_check">
       <property name="text">
        <string>二值图（带边界）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>高度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_height">
       <property name="maximum">
        <number>480</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>宽度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_width">
       <property name="maximum">
        <number>480</number>
       </property>
       <property name="value">
        <number>80</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>939</width>
        <height>392</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="MouseEvent" name="imaglabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QSlider" name="PicSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Piclabel">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btn_atlas">
     <property name="text">
      <string>选择播放图片集</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>198</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>字节数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_bufsize"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>周期</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_picT"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MouseEvent</class>
   <extends>QLabel</extends>
   <header location="global">mouseevent.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
